src/plugins/data/common/index_patterns/index_patterns/index_patterns.ts

```diff
@@ -276,11 +284,20 @@ export class IndexPatternsService {
     title: string,
     options: GetFieldsOptions
   ) => {
+    // eslint-disable-next-line no-console
+    console.log('refreshFieldSpecMap start');
     const scriptdFields = Object.values(fields).filter((field) => field.scripted);
     try {
       const newFields = await this.getFieldsForWildcard(options);
-      return this.fieldArrayToMap([...newFields, ...scriptdFields]);
+      // eslint-disable-next-line no-console
+      console.log('getFieldsForWildcard returns', JSON.stringify(newFields));
+      const result = this.fieldArrayToMap([...newFields, ...scriptdFields]);
+      // eslint-disable-next-line no-console
+      console.log('fieldArrayToMap returns', JSON.stringify(result));
+      return result;
     } catch (err) {
+      // eslint-disable-next-line no-console
+      console.log('refreshFieldSpecMap ERROR', JSON.stringify(err));
```

Comment:
@mattkime the error, when it fails is an empty object , it's not very useful 

```
[00:10:53]               │ debg browser[INFO] http://localhost:6191/37923/bundles/plugin/data/data.plugin.js 0:594180 "refreshFieldSpecMap start"
[00:10:53]               │ debg browser[INFO] http://localhost:6191/37923/bundles/plugin/data/data.plugin.js 0:594699 "refreshFieldSpecMap ERROR" "{}"
```
so the `getFieldsForWildcard` seems to fail without any description why, do you have an idea?

Reply:
No idea, first I've seen this. It would be good to add debugging as close to the call as possible just to make sure something silly isn't happening. Past that, perhaps we can determine if its a particular class of error that we can appropriately handle.